gui/control-panel.lua

```diff
@@ -140,6 +140,9 @@ local REPEATS = {
     ['empty-wheelbarrows']={
         desc='Empties wheelbarrows which have rocks stuck in them.',
         command={'--time', '1', '--timeUnits', 'days', '--command', '[', 'fix/empty-wheelbarrows', '-q', ']'}},
+    ['corrupt-jobs']={
+        desc='Removes corrupt jobs from affected units.',
+        command={'--time', '1', '--timeUnits', 'days', '--command', '[', 'fix/corrupt-jobs', ']'}},
```

Comment:
now that we found a [fixed the genesis of the problem](https://github.com/DFHack/dfhack/pull/3877), I don't think we need to add it to the control panel. Instead, it should auto-run on fort load once to clean up the effects of existing errors